tox.ini

```diff
@@ -84,16 +80,15 @@ commands =
 
 [testenv:mypy-common]
 basepython = python3
+deps = -rdev_requirements/linter-requirements.txt
+
+[testenv:mypy-py3]
+basepython = {[testenv:mypy-common]basepython}
 deps =
-    # mypy outputs coverage data in a coverage 4.x format
     coverage~=4.0
     mypy==0.812
     mypy_extensions==0.4.3
     typing>=3.6.2
```

Comment:
Do these need to be moved somewhere that dependabot can update?

Reply:
I just thought of a way to do this, I was running into a wall because of the conflicting dependencies in `linter-requirements.txt` it would not install the correct versions. Potential fix coming in!